grpc-proto/proto/stargate.proto

```diff
@@ -29,5 +30,11 @@ service Stargate {
 
   // Executes a batch of CQL queries.
   rpc ExecuteBatch(Batch) returns (Response) {}
+
+  // Schema creation
+  rpc ExecuteSchemaQuery(CqlSchemaQuery) returns (Response) {}
```

Comment:
I know Cassandra calls all kinds of operations "queries", so maybe we want to be compatible with that.
But outside C*, queries are for reading/requesting things, not modifications. So wondering if semantically it'd be better to use "SchemaOperation" / "SchemaChange" or something like that?

Reply:
Good thought, I'll change to `SchemaOperation`